### PR TITLE
BROKERS: Initialize Storage Broker #20

### PR DESCRIPTION
**Pull request by ADILATIC**
Closes: #20

### CODE REVIEW

**Comment by glhays** [comment]
@ADILATIC Name your Pull Request appropriately.
See The Standard Team for naming and branching.

**Comment by glhays** [comment]
@ADILATIC Add the closes issue # in your Pull Request comment

File: InternTrack.Core.Api/Brokers/Storages/StorageBroker.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    public class StorageBroker: EFxceptionsContext, IStorageBroker 
+    {
+      private readonly IConfiguration configuration;
+        
+       public StorageBroker(IConfiguration configuration)
+        {
+            this.configuration = configuration;
+            this.Database.Migrate();
+        }
+
```
Comment (hassanhabib) [inline InternTrack.Core.Api/Brokers/Storages/StorageBroker.cs:20]: Remove extra line here

**Comment by ADILATIC** [comment]
Closes: #20

**Review by hassanhabib — APPROVED** [review]

**Pull request merged**